.env.example

```diff
@@ -2,3 +2,6 @@ SECRET_KEY = "YOUR APP SECRET KEY GOES HERE"
 
 # database connection URL in format {provider}://{user}:{password}@{host}:{port}/{db}
 DATABASE_URL = "postgresql://janedoe:mypassword@localhost:5432/mydb"
+
+# database connection URL for tests
+DATABASE_URL_TEST = "postgresql://janedoe:mypassword@localhost:5432/testdb"
```

Comment:
this environment variable is not necessary, we can keep `DATABASE_URL` and adjust it to "prod" and tests database URLs